src/cascadia/TerminalApp/TerminalWindow.cpp

```diff
@@ -872,6 +872,7 @@ namespace winrt::TerminalApp::implementation
                         if (!focusedObject)
                         {
                             focusedObject = _root.try_as<IInspectable>();
+                            break;

```

Comment:
But... what's the point of assigning to `focusedObject` now? Doesn't this break this new code for Alt+Space?

Reply:
alt-space seems to work on the build we just ran... huh

Reply:
wait what is going on, that totally fixed it before. Hmmmmmmmm

Reply:
WHAT 
 
 
Open Terminal Stable.
Go to the sui. Make sure you can scroll the page.
Tap alt. 
Bye bye _scrolling_

Reply:
_wtf_

Reply:
okay _what_

There's something REAL fucky with the direct key handler, on the TerminalPage. Cause:
* Open the right click context menu
* tap alt
* press arrow key:
* a wild system menu appears